config/initializers/carrierwave.rb

```diff
@@ -0,0 +1,36 @@
+# TODO: what happens if the ENVs are missing?
+# TODO: is ENV safe way to handle these credentials?
```

Comment:
So, I noticed you have upgraded to rails 5.1. Is ENV still the perferred way, or should I use the `rails secrets` command now? I don't know enough to know what is better/more secure

```sh
website$ rails secrets:setup --help
Usage:
  bin/rails secrets [options]

=== Storing Encrypted Secrets in Source Control

The Rails `secrets` commands helps encrypting secrets to slim a production
environment's `ENV` hash. It's also useful for atomic deploys: no need to
coordinate key changes to get everything working as the keys are shipped
with the code.

[...]
```